libfabric.map.in

```diff
@@ -7,6 +7,7 @@ FABRIC_1.0 {
 		fi_version;
 		fi_strerror;
 		fi_tostr;
+		fi_tostr_r;
```

Comment:
This is new call, not a 1.0 ABI function.  This should be added to the end of the 1.3 list, or maybe added as a new 1.4 call... I need to check on that.

@jsquyres - Do you know off the top of your head how to handle exposing a new function?  Do we just add this to the 1.3 section, or is a 1.4 section needed, and we need to bump the current abi?

Reply:
I believe that we have to add a 1.4 section (similar to the other 1.x sections).  It can basically "inherit" from 1.3 and add one new symbol to that.

Reply:
That sounds right, but do you know where there's documentation for this?  I've been searching for a while, and I haven't been able to find anything useful.

Reply:
![magic](https://user-images.githubusercontent.com/617316/84197206-8b193c00-aa6f-11ea-8e74-b5e30c689dcc.gif)

Reply:
Magic == https://www.gnu.org/software/gnulib/manual/html_node/LD-Version-Scripts.html

Reply:
Sorry, I think that this is the definitive doc: https://sourceware.org/binutils/docs/ld/VERSION.html#VERSION

Reply:
Thanks Sir Jeff.  The documentation isn't going out of its way to be understandable.  But based on the link Jeff provided, and also this one (using terms from the above link to help google):
https://gcc.gnu.org/wiki/SymbolVersioning

- Add a 1.4 section to the map.in file with the new call
- functions marked with CURRENT_SYMVER need to move to DEFAULT_SYMVER(... 1.3)
- The new call should be listed as CURRENT_SYMVER

Compiled applications built against 1.10 library should work with the new one.
Compiled applications built against these changes that do not call the new function should work against 1.10.
Compiled applications built against these changes that call the new function should fail against 1.10.

Please verify those scenarios.

Reply:
I *think* that is right.  But it should be tested.

Reply:
Thanks for your review comments, guys! I'll be addressing all your comments very soon.